sample/library/build.gradle

```diff
@@ -10,7 +10,8 @@ dependencies {
   kapt deps.dagger2.compiler
 }
 
-compileKotlin {
+//noinspection UnnecessaryQualifiedReference
+tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).configureEach {
```

Comment:
Missing a `jvmTarget = JavaVersion.VERSION_1_8` inside this block?

Reply:
I do this in the root project for all subprojects. So it's not necessary here.
```
  tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
    kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8
  }
```